src/themes/foundation/bootstrap-table-foundation.js

```diff
@@ -57,7 +57,12 @@ $.BootstrapTable = class extends $.BootstrapTable {
   initPagination () {
     super.initPagination()
 
-    if (this.options.pagination && !this.options.onlyInfoPagination) {
+    let paginationParts = this.options.paginationParts
+    if (typeof paginationParts === 'string') {
+      paginationParts = paginationParts.replace(/\[|\]| /g, '').toLowerCase().split(',')
+    }
+
+    if (this.options.pagination && paginationParts.includes('pagesize')) {
```

Comment:
Same here

Reply:
I was not done yet :P
I waited for the answer of my question above, i will resolve the requested Changes now :-)